build/pulsar_ci_tool.sh

```diff
@@ -154,6 +154,59 @@ function ci_move_test_reports() {
   )
 }
 
+function ci_check_ready_to_test() {
+  if [[ -z "$GITHUB_EVENT_PATH" ]]; then
+    >&2 echo "GITHUB_EVENT_PATH isn't set"
+    return 1
+  fi
+  # check ready-to-test label
+  if ! yq eval -e '.pull_request.labels[] | .name | select(. == "ready-to-test")' "$GITHUB_EVENT_PATH" &> /dev/null; then
```

Comment:
Out of curiosity, I suppose `GITHUB_EVENT_PATH` represents a JSON file, so `yq` can process JSON files also?

Reply:
Yes. "man yq" says "Keep  in mind JSON is a subset of YAML." . 
There's not really a need to use yq here, it could be replaced with jq to be consistent.

Reply:
replaced with jq